samples/openthread/cli/harness/nRF_Connect_SDK.py

```diff
@@ -271,7 +271,7 @@ def getVersionNumber(self):
     def log(self, fmt, *args):
         try:
             msg = fmt % args
-            logging.info('%s - %s', self.port, msg)
+            logging.info('%s - %s', self, msg)
```

Comment:
What is the benefit of this change? There is no `__str__` defined for this class. It will not be more descriptive, will it?

Reply:
This change come from alignment with upstream THCI file. There is no `__str__` but there is `__repr__`.